css/style.css

```diff
@@ -0,0 +1,167 @@
+<<<<<<< HEAD
+main {
+    display: flex;
```

Comment:
Kindly add linters to this project. There are too many indentation errors in your CSS. Linters will make it easier to identify them

Reply:
- [ ] Good observation. Linters indeed help us write code of good quality with consistency across the team 🍔